frameworks/Rust/xitca-web/xitca-web-wasm.dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM rust:1.67 AS compile
+FROM rust:latest AS compile
```

Comment:
Please keep this locked down to a specific version

Reply:
fixed.